
CondaError: Run 'conda init' before 'conda activate'

/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py:345: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  model_reload.load_state_dict(torch.load(os.path.join(log_directory, 'best_model.pth')))
/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py:345: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  model_reload.load_state_dict(torch.load(os.path.join(log_directory, 'best_model.pth')))
/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py:345: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  if model_type == 'UNet2':
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 244, in run
    self._run()
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 275, in _run
    self._record_writer.write(data)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/record_writer.py", line 40, in write
    self._writer.write(header + header_crc + data + footer_crc)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/compat/tensorflow_stub/io/gfile.py", line 775, in write
    self.fs.append(self.filename, file_content, self.binary_mode)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/compat/tensorflow_stub/io/gfile.py", line 167, in append
    self._write(filename, file_content, "ab" if binary_mode else "a")
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/compat/tensorflow_stub/io/gfile.py", line 171, in _write
    with io.open(filename, mode, encoding=encoding) as f:
FileNotFoundError: [Errno 2] No such file or directory: b'results/testing/larger_area_048/UNet6_150_0.0001_64/events.out.tfevents.1730913769.gpu006.cluster.176035.0'
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing.py", line 94, in <module>
    print('data testing folder already prepared')
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 272, in hyperparam_tuning_training
    writer = SummaryWriter(log_dir=log_directory)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 58, in train_model
    writer.add_scalar('Loss/validation_epoch', running_val_loss / len(validation_loader), epoch)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/torch/utils/tensorboard/writer.py", line 381, in add_scalar
    self._get_file_writer().add_summary(summary, global_step, walltime)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/torch/utils/tensorboard/writer.py", line 115, in add_summary
    self.add_event(event, global_step, walltime)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/torch/utils/tensorboard/writer.py", line 99, in add_event
    self.event_writer.add_event(event)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 117, in add_event
    self._async_writer.write(event.SerializeToString())
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 171, in write
    self._check_worker_status()
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 212, in _check_worker_status
    raise exception
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 244, in run
    self._run()
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/event_file_writer.py", line 275, in _run
    self._record_writer.write(data)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/summary/writer/record_writer.py", line 40, in write
    self._writer.write(header + header_crc + data + footer_crc)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/compat/tensorflow_stub/io/gfile.py", line 775, in write
    self.fs.append(self.filename, file_content, self.binary_mode)
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/compat/tensorflow_stub/io/gfile.py", line 167, in append
    self._write(filename, file_content, "ab" if binary_mode else "a")
  File "/eejit/home/hausw001/.conda/envs/env_hybGGM/lib/python3.10/site-packages/tensorboard/compat/tensorflow_stub/io/gfile.py", line 171, in _write
    with io.open(filename, mode, encoding=encoding) as f:
FileNotFoundError: [Errno 2] No such file or directory: b'results/testing/larger_area_048/UNet6_150_0.0001_64/events.out.tfevents.1730913769.gpu006.cluster.176035.0'
